dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -817,12 +817,12 @@ public void testWorkflowFreezing() throws ApiException {
      */
     @Test
     public void testFreezingInvalidWorkflow() {
-        String versionToSnapshot = "1.0.1";
+        String versionToSnapshot = "1.0";
         ApiClient client = getWebClient(USER_2_USERNAME, testingPostgres);
         WorkflowsApi workflowsApi = new WorkflowsApi(client);
-        Workflow workflow = manualRegisterAndPublish(workflowsApi, "dockstore-testing/md5sum-checker", "",
+        Workflow workflow = manualRegisterAndPublish(workflowsApi, "dockstore-testing/tagged-apptool", "",
```

Comment:
A  little unclear by looking at just this repo, why is this repo/workflow a replacement rather than adding a new test and/or extending this test to cover both?

Reply:
The snapshot code uses `CWLHandler.getContent()` to determine the dockerPulls to image, and now, `getContent()` will find the dockerPull in the descriptor (rather than failing and returning no dockerPulls, as before).  This wouldn't have been a problem, except that the repo `md5sum-checker` references a quay image without a tag, so this snapshotting and this test failed, because the snapshot code wants the quay reference to contain a non-latest tag.  Repo `tagged-apptool` has a tagged branch containing a tool with a tagged quay reference.  I created it because it seemed less risky than changing an existing repo as necessary.  Turns out I could use it for the new IT, also.